spec/spec_helper.rb

```diff
@@ -3,10 +3,8 @@
 require 'tmpdir'
 
 RSpec.configure do |config|
-  if RSpec::Rerun.rspec3?
-    config.expect_with :rspec do |c|
-      c.syntax = [:should, :expect]
-    end
+  config.expect_with :rspec do |c|
```

Comment:
I think we can get rid of this now that it's only RSpec 3.x, and convert everything with transpec into RSpec 3.x `expect` syntax.